app/pulp/app/response.py

```diff
@@ -0,0 +1,32 @@
+from gettext import gettext as _
+
+from rest_framework.response import Response
+
+
+class OperationPostponedResponse(Response):
+    """
+    An HTTP response class for returning 202 and a list of spawned tasks.
+
+    This reponse object should be used by views that dispatch asynchronous tasks. Most common use
+    case is for sync and publish operations. When the task_group_id is specified, the task_results
```

Comment:
We can comment on the task group use case on the redmine story. @dkliban can you link us to it when that is in posted?